pcsx2/Pcsx2Config.cpp

```diff
@@ -264,20 +264,14 @@ const char* Pcsx2Config::GSOptions::GetRendererName(GSRendererType type)
 {
 	switch (type)
 	{
-		case GSRendererType::Auto:
-			return "Auto";
-		case GSRendererType::DX11:
-			return "Direct3D 11";
-		case GSRendererType::OGL:
-			return "OpenGL";
-		case GSRendererType::VK:
-			return "Vulkan";
-		case GSRendererType::SW:
-			return "Software";
-		case GSRendererType::Null:
-			return "Null";
-		default:
-			return "";
+	case GSRendererType::Auto: return "Auto";
+	case GSRendererType::DX11: return "Direct3D 11";
+	case GSRendererType::Metal: return "Metal";
+	case GSRendererType::OGL: return "OpenGL";
+	case GSRendererType::VK: return "Vulkan";
+	case GSRendererType::SW: return "Software";
+	case GSRendererType::Null: return "Null";
+	default: return "";
```

Comment:
Might be worthwhile adding `// clang-format off` and `// clang-format on` here if you don't like what clang format does

Reply:
With the frequency of these single-item switch statements, I feel like having to add the clang format comments is kind of ridiculous

If you use `git clang-format` (which you should), this only becomes an issue if you actually edit the switch statement, since the braces have the right indentation which should protect it from any outside changes bleeding in

I should probably format that file nicely though, instead of just reverting the clang-format change and leaving it somewhere in the middle